ci-operator/jobs/medik8s/node-healthcheck-operator/medik8s-node-healthcheck-operator-main-presubmits.yaml

```diff
@@ -75,23 +75,21 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
     branches:
-    - ^master$
-    - ^master-
-    cluster: build02
+    - ^main$
+    - ^main-
+    cluster: build01
     context: ci/prow/images
     decorate: true
     decoration_config:
       skip_cloning: true
     labels:
       ci.openshift.io/generator: prowgen
-      job-release: "4.8"
+      job-release: "4.9"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-medik8s-node-healthcheck-operator-master-images
-    optional: true
+    name: pull-ci-medik8s-node-healthcheck-operator-main-images
     rerun_command: /test images
-    skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
why we removed this? I think Roy added this on purpose to avoid tests for non-code changes

Reply:
Oh, missed this, good catch.
This file is generated. Will check how I can add it to the config file.